src/components/ha-yaml-editor.ts

```diff
@@ -41,7 +41,7 @@ export class HaYamlEditor extends LitElement {
     try {
       this._yaml =
         value && !isEmpty(value)
-          ? dump(value, { schema: this.yamlSchema })
+          ? dump(value, { schema: this.yamlSchema, quotingType: '"' })
```

Comment:
I kept/added the quoting type here, to be closer to our YAML guidelines (and match the documentation as well).